src/palace/manager/celery/tasks/reports.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import annotations
+
+from celery import shared_task
+from typing_extensions import Unpack
+
+from palace.manager.celery.task import Task
+from palace.manager.reporting.reports.library_collection import (
+    LibraryCollectionReport,
+    LibraryReportKwargs,
+    LibraryTitleLevelReport,
+)
+from palace.manager.service.celery.celery import QueueNames
+
+REPORT_KEY_MAPPING: dict[str, type[LibraryCollectionReport]] = {
```

Comment:
Minor: Might be nice to use a `frozendict` here. Its nice to have these global variables be immutable data structures.